lib/engine/game/g_18_eus/step/bidbox_auction.rb

```diff
@@ -5,128 +5,130 @@
 module Engine
   module Game
     module G18EUS
-      module BidboxAuction
-        include Engine::Step::Auctioner
+      module Step
+        module BidboxAuction
+          include Engine::Step::Auctioner
+
+          def actions(entity)
+            actions = []
+            actions << 'bid' << 'pass'
+            actions.concat(super).uniq
```

Comment:
There is no logic, so its `%w[bid pass].concat(super).uniq`

Reply:
Changed